xls/integrator/ir_integrator.cc

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 Google LLC
```

Comment:
How about putting this in xls/contrib/integrator? Once its more developed and more generally useful we can move it to the main directory. As is, the use case is pretty specific.

Reply:
Done.